applications/nrf5340_audio/README.rst

```diff
@@ -1227,6 +1241,36 @@ Testing the BIS mode is identical to `Testing the default CIS mode`_, except for
 
 .. _nrf53_audio_app_porting_guide:
 
+Testing FOTA upgrades
+---------------------
+
+`nRF Connect Device Manager`_ can be used for testing FOTA upgrades.
+The procedure for upgrading the firmware is identical for both headset and gateway firmware.
+You can test upgrading the firmware on both cores at the same time on a gateway device by completing the following steps:
+
+1. Make sure you have :ref:`done the necessary configurations for FOTA. <nrf53_audio_app_configuration_configure_fota>`
+#. Install `nRF Connect Device Manager`_ on your Android or iOS device.
+#. Connect an external flash shield to the gateway.
+#. Make sure the gateway runs a firmware that supports DFU using external flash memory.
+#. Use the :file:`buildprog.py` script to create a zip folder that contains new firmware for both cores:
+
+   .. code-block:: console
+
+      python buildprog.py -c both -b debug -d gateway --pristine -m external
+
+   .. note::
+      If you use Linux, the ``python`` command must excute Python 3.
+
+#. Transfer the generated zip file to your Android or iOS device.
+   The file name should start with :file:`dev_gateway_build_debug_dfu_application`.
+   For transfer, you can use cloud services like Google Drive for Android or iCloud for iOS.
+#. Open `nRF Connect Device Manager`_ and look for ``NRF5340_AUDIO_GW_DFU`` in the scanned devices window.
```

Comment:
Did we mention how to get into DFU mode on the device itself?

Reply:
This should be covered by step 4, I expanded this step